js/ui/windowManager.js

```diff
@@ -1579,8 +1579,10 @@ const WindowManager = new Lang.Class({
                          });
         } else {
             let monitor = Main.layoutManager.monitors[actor.meta_window.get_monitor()];
-            let xDest = monitor.x + actor.x;
```

Comment:
The removal of xDest and yDest should be moved to the next commit.

Reply:
Oops! Late-evening rebasing after long days is not a good idea :). Fixed